rasa/core/actions/forms.py

```diff
@@ -98,11 +98,14 @@ def get_mappings_for_slot(
 
         If None, map requested slot to an entity with the same name
         """
-        requested_slot_mappings = self._to_list(
-            domain.slot_mapping_for_form(self.name()).get(
-                slot_to_fill, self.from_entity(slot_to_fill),
-            )
-        )
+        # requested_slot_mappings = self._to_list(
+        #     domain.slot_mapping_for_form(self.name()).get(
+        #         slot_to_fill, self.from_entity(slot_to_fill),
+        #     )
+        # )
+        domain_slots = domain.as_dict().get("slots")
```

Comment:
not related but why do we need `as_dict()`? Can't we just access the things directly on the `domain` object?

Reply:
I couldn't see another way to access the slot mappings for a specific slot_name - `domain.slots` is a list of `Slot` objects and I couldn't see how to 'fish' the right Slot 🤔

Reply:
mhm, whatever `as_dict` contains must be _somewhere_ in the domain object.